[0;33m--- WARNING: GDB cannot open serial ports accessed as COMx[0m
[0;33m--- Using \\.\COM5 instead...[0m
[0;33m--- esp-idf-monitor 1.2.1 on \\.\COM5 115200 ---[0m
[0;33m--- Quit: Ctrl+] | Menu: Ctrl+T | Help: Ctrl+T followed by Ctrl+H ---[0m
[0;33m0x4080600a: rv_utils_wait_for_intr at C:/Users/Pedro/esp/esp-idf/components/riscv/include/riscv/rv_utils.h:33
 (inlined by) esp_cpu_wait_for_intr at C:/Users/Pedro/esp/esp-idf/components/esp_hw_support/cpu.c:119
[0m
[0;33mStack dump detected[0m
[0;33m0x4080062e: panic_abort at C:/Users/Pedro/esp/esp-idf/components/esp_system/panic.c:452
[0m
[0;33m0x408085dc: __ubsan_include at C:/Users/Pedro/esp/esp-idf/components/esp_system/ubsan.c:313
[0m
[0;33m0x40800001: _vector_table at ??:?
[0m
[0;33mFailed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Pedro\\esp\\astromsg\\build\\light_bulb.elf', '-ex', 'target remote | "C:\\Users\\Pedro\\.espressif\\python_env\\idf5.1_py3.11_env\\Scripts\\python.exe" -m "esp_idf_panic_decoder" --target esp32c6 "C:\\Users\\Pedro\\AppData\\Local\\Temp\\tmpulordh3c"', '-ex', 'bt']' returned non-zero exit status 1.
b"error starting child process '| C:\\Users\\Pedro\\.espressif\\python_env\\idf5.1_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c6 C:\\Users\\Pedro\\AppData\\Local\\Temp\\tmpulordh3c': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m0x4001975a: software_reset_cpu in ROM
[0m
[0m
